CONTRIBUTING.md

```diff
@@ -41,12 +132,56 @@ source .venv/bin/activate
 pip install -r requirements-dev.txt
 ```
 
-Then, build the docs with Sphinx:
+Then, build the docs:
 
 ```sh
+python tools/concat_release_notes.py
 python -m sphinx -W docs/ docs/_build
```

Comment:
FYI we can make this a lot simpler by adopting Tox or Poetry. I'll DM you all about this.